packages/message-parser/src/grammar.pegjs

```diff
@@ -608,7 +608,7 @@ emoticonPattern
   / e:$(":'(" / ":'-(" / ";(" / ";-(") { return emoticon(e, 'cry'); }
   / e:$(">:(" / ">:-(" / ":@") { return emoticon(e, 'angry'); }
   / e:$(":$" / "=$") { return emoticon(e, 'flushed'); }
-  / e:$"D:" { return emoticon(e, 'fearfulc'); }
+  / e:$"D:" { return emoticon(e, 'fearful'); }
```

Comment:
Could you please add a test case for this?

Reply:
Sure!